lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -233,8 +233,12 @@ module InternalConstants
     Fluent::Plugin.register_output('google_cloud', self)
 
     PLUGIN_NAME = 'Fluentd Google Cloud Logging plugin'.freeze
-    # Extract plugin version by finding the spec this file was loaded from.
     PLUGIN_VERSION = begin
+      # Extract plugin version from file path.
+      __FILE__.match(
+        %r{fluent-plugin-google-cloud-(?<version>[0-9a-zA-Z\.]*)/})['version']
+    rescue StandardError
+      # Extract plugin version by finding the spec this file was loaded from.
       dependency = Gem::Dependency.new('fluent-plugin-google-cloud')
       all_specs, = Gem::SpecFetcher.fetcher.spec_for_dependency(dependency)
       matching_spec, = all_specs.grep(
```

Comment:
[Optional] My fault — I forgot to rename `matching_spec`. Should have been `matching_version` instead.